apps/meteor/tests/end-to-end/api/emoji-custom.ts

```diff
@@ -95,7 +95,7 @@ describe('[EmojiCustom]', () => {
 				})
```

Comment:
Can you add more cases? (in case they're not there already)
- what happens when no file is provided to update
- what happens when file is provided but none of the data is updated (just the file)
- Updating the emoji data but no file
- Requesting emoji without passing the etag (for old emojis)
- Requesting an emoji with an invalid etag

Reply:
Some tests are already there.
- what happens when no file is provided to update
- Updating the emoji data but no file
These should be covered by
https://github.com/RocketChat/Rocket.Chat/blob/e0fb435ef0b89ef413145205672c5e3382649811/apps/meteor/tests/end-to-end/api/emoji-custom.ts#L99
- what happens when file is provided but none of the data is updated (just the file)
The end point is implemented in such a way that it replaces the provided data completely with the data in db. So it always expects a complete new data in body. So aliases is the only optional field required for the endpoint. This case is covered by https://github.com/RocketChat/Rocket.Chat/blob/e0fb435ef0b89ef413145205672c5e3382649811/apps/meteor/tests/end-to-end/api/emoji-custom.ts#L115